lib/grpffmat.gi

```diff
@@ -75,9 +73,7 @@ local   deg,  mat,  row,  char,  B;
     elif Is8BitMatrixRep(mat) then
       deg:=LcmInt( deg, Length(FactorsInt(Q_VEC8BIT(mat![2]))));
```

Comment:
This case seems unnecessary. The code after `else` calls an efficient method.

Reply:
The `else` method calls a default method which iterates over all rows; this check only checks a single row, under the assumption that in a `Is8BitMatrixRep` all rows are defined over the same field... I have not checked if this is even correct to do; and of course it could also be achieved by installing a dedicated `DegreeFFE ` method for `Is8BitMatrixRep`. But for now I wanted to keep the changes to a minimum to reduce the risk of breaking more than I am fixing...

Reply:
That said, I'll just add DegreeFFE methods for `IsGF2MatrixRep` and `Is8BitMatrixRep` in the next update for this PR

Reply:
Aha, but actually a dedicated `DegreeFFE` for `Is8BitMatrixRep` still would be de-optimization: because it would then end up checking all entries of the matrix to determine the smallest field; while the existing code here just looks at a single row to determine the desired field / degree (as in this function, we are not looking for the smallest field containing all elements, just *some* field)